Source/Utils/Assert.hpp

```diff
@@ -3,7 +3,9 @@
 #include "Utils/Logger.hpp"
 
 #ifdef NDEBUG
-#define Assert(expression) if (!(expression)) std::cout << "Assertion failed: " << #expression << ", file " << __FILE__ << ", line " << __LINE__ << "\n"
+#define Assert(expression)                                                                                             \
+    if (!(expression))                                                                                                 \
+    std::cout << "Assertion failed: " << #expression << ", file " << __FILE__ << ", line " << __LINE__ << "\n"
```

Comment:
lets add braces than

Reply:
hm, which braces?

Reply:
If you propose curled braces **_{}_** - they will screw last semicolon
Now we write **_Assert(x);_** and there is no **_;_** in Assert definition